src/data-pipeline/utils/utils-lambda.ts

```diff
@@ -99,7 +99,6 @@ export class LambdaUtil {
         securityGroups: [securityGroupForLambda],
         role: lambdaRole,
         entry: join(__dirname, '..', 'lambda', 'partition-syncer', 'index.ts'),
-        reservedConcurrentExecutions: 1,
```

Comment:
add/update a test case to make sure the field not filling

Reply:
Added